docs/agile-development/team-agreements/readme.md

```diff
@@ -2,10 +2,10 @@
 
```

Comment:
This file no longer appears as the first thing under Team Agreements so it looks a little funny 
<img width="257" alt="Screen Shot 2021-04-29 at 9 27 22 PM" src="https://user-images.githubusercontent.com/10625466/116637370-baf45b00-a931-11eb-9414-50fff4464fa4.png">

Reply:
renamed all readme.md to README.md and set up index pages for mkdocs - that way all readmes show up first in their respective sections (this does not work for readme.md, hence the change)